repos/fdbt-site/src/pages/api/deletePassenger.ts

```diff
@@ -0,0 +1,28 @@
+import { NextApiResponse } from 'next';
+import { deletePassengerTypeByNocCodeAndName } from '../../data/auroradb';
+import { redirectToError, redirectTo, getAndValidateNoc } from './apiUtils/index';
+import { NextApiRequestWithSession } from '../../interfaces';
+
+export default async (req: NextApiRequestWithSession, res: NextApiResponse): Promise<void> => {
+    try {
+        const { query } = req;
+
+        const name = (query?.name as string) || '';
+
+        const isGroup = (query?.isGroup as string) || '';
```

Comment:
Don't think you need the `|| ''` here as the null case is handled on line 14. Also rather than casting to a string it would be safer to do a `typeof x === 'string'` check in the check on 14 as it is possible to pass other values in the API request.

Reply:
Good catch. Updated.